ngraph/src/ngraph/op/interpolate.hpp

```diff
@@ -47,6 +47,20 @@ namespace ngraph
                 // specify the number of pixels to add to the end of the image being interpolated.
                 // This addition of pixels is done before interpolation calculation.
                 std::vector<size_t> pads_end;
+
+                InterpolateAttrs(){};
```

Comment:
What was the reason to update v0 operation but not v3?

Reply:
Fixed